src/pinns_pde_solve.jl

```diff
@@ -381,7 +398,9 @@ function DiffEqBase.solve(
     args...;
     abstol = 1f-6,
     verbose = false,
-    maxiters = 100)
+    maxiters = 100,
+    stochastic_loss = true,
```

Comment:
why is this not part of the discretization? The setup here is still really odd and should get fixed